qutebrowser/javascript/webelem.js

```diff
@@ -55,7 +55,9 @@ window._qutebrowser.webelem = (function() {
         try {
             caret_position = elem.selectionStart;
         } catch (err) {
-            if (err instanceof DOMException &&
+            if ((err instanceof DOMException ||
+                 (frame !== null &&
+                  err instanceof frame.DOMException)) &&
```

Comment:
This seems a bit complicated - what about something like this?

```js
            if (err instanceof (frame ? frame.DOMException : DOMException)) {
```

Reply:
eslint complains if I use ternary operators at all (but I do agree this is a bit of convoluted logic). Should I assign a variable above this in an if statement or should I try to silence the ternary warning?